schemas/preferences.sublime-settings.json

```diff
@@ -1045,6 +1045,11 @@
           "items": {"type": "string"},
           "default": ["*.log"]
         },
+        "update_check": {
+          "markdownDescription": "Whether update check on start is enabled.\n\n**Can only be disabled if Sublime Text is registered.**",
```

Comment:
"Whether update check on start is enabled." -> "Check for updates when the application is started."

"Can only be" -> "This can only be"